client/components/contributepage/InfoSection.js

```diff
@@ -0,0 +1,163 @@
+  margin: 64px 0;
+`;
+
+const BigItem = styled.li`
+  padding: 16px 0;
+  list-style-type: none;
+  list-style-position: inside;
+`;
+
+const ItemTitle = styled.h3`
+  display: inline-block;
+  font-size: 31px;
+  font-weight: bold;
+  margin: 16px 0;
+`;
+
+const ItemContent = styled.div`
+  font-size: 18px;
+  margin: 18px 32px;
+  font-weight: normal;
+  ol li {
+    list-style-type: decimal;
+    list-style-position: inside;
+  }
+`;
+
+const CodeBlock = styled.code`
+  white-space: pre-wrap;
+  display: inline-block;
+  background: ${props => props.theme.colors.lightGrey};
+  background: #ebeff3; /* props.theme.colors.lightGrey is currently #ccc - outside of design spec */
+  padding: 8px 24px;
+  margin: 16px 0;
+`;
+
+const InfoSection = () => {
+  return (
+    <Article>
+      <BigList>
+        <BigItem>
+          <ItemTitle>
+            Choose an{" "}
+            <a
+              href="https://github.com/chingu-voyages/moonshot-chingu-quiz/issues"
+              rel="noreferrer noopener"
+              target="_blank"
+            >
+              issue
+            </a>
+          </ItemTitle>
+          <ItemContent>
+            <p>
+              To get started, please feel free to start working on any issues we
+              have created and that are not assigned to anyone yet. If you would
+              like to pick up an issue, please leave a comment for the other
+              contributors to see that this issue is already be worked on.
+            </p>
+            <p>
+              For starters, we recommend to pick up issues labelled good first
+              issue.
+            </p>
+          </ItemContent>
+        </BigItem>
+        <BigItem>
+          <ItemTitle>Get setup</ItemTitle>
+          <ItemContent>
+            <ol>
+              <li>
+                Fork the{" "}
+                <a
+                  href="https://github.com/chingu-voyages/moonshot-chingu-quiz"
+                  rel="noreferrer noopener"
+                  target="_blank"
+                >
+                  repository
+                </a>
+              </li>
+              <li>Clone your forked repository onto your local machine</li>
+              <li>
+                inside your CLI (Command Line Interface), move into your working
+                directory
+              </li>
+              <li>
+                run `npm i` inside the root, client *and* server folder to
+                install all dependencies needed for this project
+              </li>
+              <li>
+                inside the client folder, run `npm run dev` to start the
+                development server
+              </li>
+            </ol>
+          </ItemContent>
+        </BigItem>
+        <BigItem>
+          <ItemTitle>Create a branch for your feature</ItemTitle>
+          <ItemContent>
+            <p>Below you can find an overview of the branches we are using.</p>
+            <CodeBlock>{`- master (protected)
```

Comment:
These next few lines + end of codeblock seems like they would conflict with our eslint / prettier conflicts. Has this been tested?

Reply:
I've my editor set to format on save so I don't think it conflicts...